### ISSUE TITLE
compatiblity with dh_dwz / dwz for Debian packaging (optimize DWARF debug information in ELF binaries via dwz)

### ISSUE THREAD

**Issue by adrelanos** [GrapheneOS/hardened_malloc#85]
This happened while packaging as deb package.

```
   dh_dwz
        dwz -q -- debian/hardened-malloc/usr/lib/libhardened_malloc.so
dh_dwz: dwz -q -- debian/hardened-malloc/usr/lib/libhardened_malloc.so returned exit code 1
make: *** [debian/rules:13: binary] Error 1
dpkg-buildpackage: error: fakeroot debian/rules binary subprocess returned exit status 2
I: copying local configuration
E: Failed autobuilding of package
```

```
dwz libhardened_malloc.so 
dwz: libhardened_malloc.so: .debug_info section not present
```

source code of dh_dwz:
https://github.com/Debian/debhelper/blob/master/dh_dwz

dwz man page:
https://manpages.debian.org/buster/dwz/dwz.1.en.html

No idea if fixing this makes sense.

If fixing this does not make sense, then I will disable dh_dwz.

**Comment by adrelanos** [comment]
https://github.com/GrapheneOS/hardened_malloc/pull/91 would solve this as per https://github.com/GrapheneOS/hardened_malloc/issues/85#issuecomment-515715979.

**Comment by thestinger** [comment]
Fixed per approach for #88.
